src/app/pages/login/login-form/login-form.component.spec.ts

```diff
@@ -1,45 +1,46 @@
 import { async, ComponentFixture, TestBed } from '@angular/core/testing';
 import { RouterTestingModule } from '@angular/router/testing';
-import {FormsModule, ReactiveFormsModule, NgForm } from '@angular/forms';
+import { FormsModule, ReactiveFormsModule, NgForm } from '@angular/forms';
 import { of, throwError } from 'rxjs';
 import { ToastrModule, ToastrService } from 'ngx-toastr';
 import { NgxSpinnerModule } from 'ngx-spinner';
 
 import { LoginFormComponent } from './login-form.component';
 import { LoginHeaderComponent } from '../login-header/login-header.component';
 import { LoginSliderComponent } from '../login-slider/login-slider.component';
-import { SocialLoginComponent } from '../social-login/social-login.component';
 import { HttpClientTestingModule } from '@angular/common/http/testing';
 import { LoginService } from 'src/app/services/login/login.service';
-import { loginServiceSpy, resetSpies, toastServiceSpy } from '../../../helpers/spies';
+import {
+  loginServiceSpy,
+  resetSpies,
+  toastServiceSpy
+} from '../../../helpers/spies';
```

Comment:
Please us absolute imports

Reply:
this seems to have been merged with s previous PR